auth-api/src/auth_api/resources/org.py

```diff
@@ -56,15 +55,18 @@ def post():
         valid_format, errors = schema_utils.validate(request_json, 'org')
         if not valid_format:
             return {'message': schema_utils.serialize(errors)}, http_status.HTTP_400_BAD_REQUEST
-
+        is_staff = token and 'staff' in token.get('realm_access').get('roles')
```

Comment:
Should it be 'staff_admin' here?

Reply:
Good catch..But I am removing that code since now staff has entry in user table..so we dont have to check for it..But good catch

Reply:
changed it